deploy/helm/sumologic/conf/cleanup/cleanup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Fix URL to remove "v1" or "v1/"
+export SUMOLOGIC_BASE_URL=${SUMOLOGIC_BASE_URL%v1*}
+# Support proxy for terraform
+export HTTP_PROXY=${HTTP_PROXY:=""}
+export HTTPS_PROXY=${HTTPS_PROXY:=""}
+export NO_PROXY=${NO_PROXY:=""}
+
+cd /cleanup/ || exit 1
+
+readonly COLLECTOR_NAME="{{- if .Values.sumologic.collectorName }}{{ .Values.sumologic.collectorName }}{{- else}}{{ .Values.sumologic.clusterName }}{{- end}}"
```

Comment:
Is this logic used somewhere else as well? If yes can we move it to the `_helpers`?

Reply:
done